app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -226,6 +226,59 @@ def test_check_live_end_requested_with_reject_review(self):
         self.assertFalse(latest_change.experiment_data["is_end_requested"])
         self.assertEqual(latest_change.message, f"Rejected: {rejection_message}")
 
+    def test_check_live_end_requested_with_reject_review_and_with_review_pushes(self):
```

Comment:
I can't quite parse this test name....  Maybe another name or a docstring?

Reply:
Hah, fair enough, updated.